Spigot-API-Patches/0159-Implement-getters-and-setters-for-EntityItem-owner-a.patch

```diff
@@ -0,0 +1,57 @@
+From ef1d47b53af89c317361d51869ac08c38ec3e380 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Sat, 6 Oct 2018 20:54:13 -0500
+Subject: [PATCH] Implement getters and setters for EntityItem owner and
+ thrower
+
+
+diff --git a/src/main/java/org/bukkit/entity/Item.java b/src/main/java/org/bukkit/entity/Item.java
+index 2e98e4f9..439cf3ab 100644
+--- a/src/main/java/org/bukkit/entity/Item.java
++++ b/src/main/java/org/bukkit/entity/Item.java
+@@ -2,6 +2,8 @@ package org.bukkit.entity;
+ 
+ import org.bukkit.inventory.ItemStack;
+ 
++import java.util.UUID;
```

Comment:
drop or comment

Reply:
was told by @aikar not to comment imports